setup.py

```diff
@@ -11,6 +11,7 @@
     'numpy>=1.13.1',
     'click>=6.7',
     'bs4>=0.0.1',
+    'finetune>=0.1.0'
```

Comment:
Is finetune already on pypy?

Reply:
Is this not redundant, can you not just load the requirements txt?

Reply:
Yeah there is a 0.1.0 release up that should function on PyPI.

Yes you can just load the requirements.txt but... there's some funkyness around actually making sure requirements.txt ships with the package to PyPI that I need to sort out.  I think it requires specifying requirements.txt as a external data file in setup.py.